tests/integration/test_zosmf.py

```diff
@@ -17,3 +17,8 @@ def test_get_info_should_return_valid_response(self):
         """Executing the get_info method should return a valid response."""
         command_output = self.zosmf.get_info()
         self.assertIsInstance(command_output, dict)
+
+    def test_list_systems_should_return_valid_response(self):
```

Comment:
@t1m0thyj I have added the `integerated test` can you test that?

Reply:
Thanks for adding, it passes 👍